src/styles/blocks/variables.scss

```diff
@@ -0,0 +1,18 @@
+$day-color: #eee;
+$hover-day-color: pink;
+$day-size: 100px;
+$day-border: 1px solid black;
+$font-family: Arial, Helvetica, sans-serif;
+$font-size: 30px;
+$grid-gap: 1px;
+$calendar-inline-padding: 10px;
+$hover-shift: -20px;
+$transition-duration: 0.5s;
+$days: "sun" 0,
```

Comment:
The month should start at the correct column (Monday is the 1st, Friday is the 5th) - in a normal world, the countdown starts from 1)0